badge_server/main.py

```diff
@@ -280,22 +286,30 @@ def _get_badge_status(
     The badge status will determine the right hand text and the color of
```

Comment:
I don't think that the return type is a string anymore

Reply:
Can I actually do this in a follow up PR?

Reply:
Done!